src/sentry/utils/services.py

```diff
@@ -321,6 +321,22 @@ def call_backend_method(context, backend):
                 context.backends[base] = backend
                 try:
                     return getattr(backend, attribute_name)(*args, **kwargs)
+                except Exception as e:
+                    # If this isn't the primary backend, we log any unexpected
+                    # exceptions so that they don't pass by unnoticed. (Any
+                    # exceptions raised by the primary backend aren't logged
+                    # here, since it's assumed that the caller will log them
+                    # from the calling thread.)
+                    if not is_primary:
+                        expected_raises = getattr(base_value, '__raises__', [])
+                        if not expected_raises or not isinstance(e, tuple(expected_raises)):
+                            logger.warning('%s caught in executor while calling %r on %s.',
```

Comment:
Right now this doesn't get picked up anything in our production configuration (the level is too low), I'll need to work with @JTCunning to get that sorted so he doesn't get mad